Libraries/PeriphDrivers/Source/GPIO/gpio_me30.c

```diff
@@ -100,50 +100,36 @@ int MXC_GPIO_Config(const mxc_gpio_cfg_t *cfg)
     }
 
     // Configure the pad
-    // TODO(ME30): "ps" (weak vs strong pull-up/down select) register field missing
     switch (cfg->pad) {
```

Comment:
Should be removed.
Duplicate line
![image](https://github.com/analogdevicesinc/msdk/assets/46590392/4c754606-3d43-4131-a1b8-ab4185836cdf)

Reply:
whoops done, thanks